packages/grafbase-graphiql/src/utils/validateQuery.ts

```diff
@@ -0,0 +1,11 @@
+import { Source, parse } from 'graphql'
+
+export const validateQuery = (source?: string | Source | null) => {
+  try {
+    if (source) {
+      return !!parse(source)
+    }
+  } catch {}
+
+  return false
+}
```

Comment:
This function is only used once, wouldn't be easier/cleaner to just do it inline (As it is just 1 line of code, parse(X) instead of moving it here, declaring types, try/catch etc..?